src/test/resources/config.json

```diff
@@ -13,9 +13,14 @@
   },
   "browserPreferences": {
     "firefox": {
-      "browser.download.defaultFolder": "downloads/reports",
+      "browser.download.dir": "downloads/reports",
       "pdfjs.disabled": true,
       "browser.download.folderList": 2
+    },
+    "chrome": {
+      "profile.default_content_settings.popups": 0,
+      "download.default_directory": "downloads/reports",
```

Comment:
for firefox we have: "browser.download.dir" and for chrome we have "download.default_directory".  Why?  Is it because thats what the drivers need and we're making it easier for users to not have to translate into our own references or is this a mistake?

Reply:
this is browser specific property for default file download directories